RNAndroidPlayground/app/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply from: "../dependencies.gradle"
+
+project.ext.vectoricons = [
+        iconFontNames: ['MaterialIcons.ttf']
+]
+
+apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.kiwi.rnandroidplayground"
+        minSdkVersion 19
+        targetSdkVersion 27
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        multiDexEnabled true
+        ndk {
+            abiFilters "armeabi-v7a", "x86"
+        }
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility 1.8
+        targetCompatibility 1.8
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    // Uncomment next line to develop locally & run "yarn start"
+    // implementation project(':rnkiwimobile')
```

Comment:
Does this mean I have to uncomment this line whenever I run locally, and remember to not commit it?

Reply:
I plan to change this in subsequents PRs. You still can run it locally (the way it is). I just didn't want to add more work into this PR. But for now if you want to run it with `implementation project` that's the way to go. Commiting it by mistake would not break anything though.

Reply:
> I plan to change this in subsequents PRs

That is good enough for me.